packages/teleport/src/Audit/fixtures/index.ts

```diff
@@ -2461,6 +2461,178 @@ export const events = [
     token_name: '************************a2418147',
     uid: 'c1ea0e6c-ee3a-4f7e-9a98-9df283b01a98',
   },
+  {
+    cluster_name: 'im-a-cluster-name',
+    code: 'TV001I',
+    device: {
+      asset_tag: 'M2CQVQV64R',
+      device_id: '99d39707-efdd-436c-94f3-6a1aeef1fbf2',
+      os_type: 2,
+    },
+    ei: 0,
+    event: 'device',
+    status: {
+      success: true,
+    },
+    time: '2023-01-12T19:28:36.842Z',
+    uid: '94d33b77-82cd-4558-8893-0320699bf755',
+    user: {
+      user: '3827e8ad-7cbe-4423-a80f-dfc89e83eb86.im-a-cluster-name',
+    },
+  },
+  {
+    cluster_name: 'im-a-cluster-name',
+    code: 'TV001I',
+    device: {
+      asset_tag: 'M2CQVQV64R',
+      device_id: '99d39707-efdd-436c-94f3-6a1aeef1fbf2',
+      os_type: 2,
+    },
+    ei: 0,
+    event: 'device',
+    status: {
+      success: false, // failed
+    },
+    time: '2023-01-12T19:28:36.842Z',
+    uid: '94d33b77-82cd-4558-8893-0320699bf755',
+    user: {
+      user: '3827e8ad-7cbe-4423-a80f-dfc89e83eb86.im-a-cluster-name',
+    },
+  },
+  {
+    cluster_name: 'im-a-cluster-name',
+    code: 'TV002I',
+    device: {
+      device_id: '99d39707-efdd-436c-94f3-6a1aeef1fbf2',
+    },
+    ei: 0,
+    event: 'device',
+    status: {
+      success: true,
+    },
+    time: '2023-01-12T20:33:20.527Z',
+    uid: 'a12e693e-1c45-43e4-a9d1-5fd8399e303c',
+    user: {
+      user: '3827e8ad-7cbe-4423-a80f-dfc89e83eb86.im-a-cluster-name',
+    },
+  },
+  {
+    cluster_name: 'im-a-cluster-name',
+    code: 'TV002I',
+    device: {
+      device_id: '99d39707-efdd-436c-94f3-6a1aeef1fbf2',
+    },
+    ei: 0,
+    event: 'device',
+    status: {
+      success: false,
+    },
+    time: '2023-01-12T20:33:20.527Z',
+    uid: 'a12e693e-1c45-43e4-a9d1-5fd8399e303c',
+    user: {
+      user: '3827e8ad-7cbe-4423-a80f-dfc89e83eb86.im-a-cluster-name',
+    },
+  },
+  {
+    cluster_name: 'im-a-cluster-name',
+    code: 'TV003I',
+    device: {
+      device_id: '99d39707-efdd-436c-94f3-6a1aeef1fbf2',
+    },
+    ei: 0,
+    event: 'device',
+    status: {
+      success: true,
+    },
+    time: '2023-01-12T19:51:54.168Z',
+    uid: '24cce2a0-57b7-494e-a196-c7fd2482b10c',
+    user: {
+      user: '3827e8ad-7cbe-4423-a80f-dfc89e83eb86.im-a-cluster-name',
+    },
+  },
+  {
+    cluster_name: 'im-a-cluster-name',
+    code: 'TV003I',
+    device: {
+      device_id: '99d39707-efdd-436c-94f3-6a1aeef1fbf2',
+    },
+    ei: 0,
+    event: 'device',
+    status: {
+      success: false,
+    },
+    time: '2023-01-12T19:51:54.168Z',
+    uid: '24cce2a0-57b7-494e-a196-c7fd2482b10c',
+    user: {
+      user: '3827e8ad-7cbe-4423-a80f-dfc89e83eb86.im-a-cluster-name',
+    },
+  },
```

Comment:
Add an entry for TV004I on the list?

I realize I'm issuing it at the moment, but that's easy enough to fix. It should look similar to TV005I (aka the enroll event).

```suggestion
  },
  {
    cluster_name: 'im-a-cluster-name',
    code: 'TV004I',
    device: {
      asset_tag: 'M2CQVQV64R',
      device_id: '0e288b23-f99f-4635-b182-06e9308095a8',
      os_type: 2,
    },
    ei: 0,
    event: 'device',
    status: {
      success: true,
    },
    time: '2023-01-12T21:31:29.191Z',
    uid: 'bbbc496f-820b-4f49-ae0d-1c1b29faee85',
    user: {
      user: 'lisa',
    },
  },
```